### DISCUSSION TITLE
Bed heating turning off on new 4.2.7 board

### DISCUSSION THREAD

**Discussion by Onejk1** [Jyers/Marlin#1434] [Q&A] [answered]
Hi. I managed to damage my 4.2.2 board when reversing the extruder direction for duel gear extruder. So I bought a 4.2.7 board and flashed the 4.2.7 UBL 15x15 HS firmware. All seemed good but it keeps turning the bed heating off after the first few layers. Could new boards have an eco mode turned on by default? I see no option to turn it off on the screen

**Comment by Jyers** [comment, marked as answer]
Most likely a hardware issue of some kind, I hope you can get it resolved
